dash.el

```diff
@@ -1858,7 +1867,7 @@ Key-value stores are disambiguated by placing a token &plist,
       (let ((s (car match-form)))
         (cons (list s source)
               (dash--match (cddr match-form) s))))
-     ((memq (car match-form) '(&keys &plist &alist &hash))
+     ((functionp (intern (format "dash-expand:%s" (car match-form))))
```

Comment:
@Fuco1 	Turn the `intern` logic into a helper function, reuse it also above in `kv-1`

Reply:
Will do.